### PR TITLE
[tf.keras] Bug - Fix Stateful Metrics in fit_generator with TensorBoard

### PR DESCRIPTION
**Pull request by brge17**
**Description:**

This mirrors the changes in ``keras:master``: https://github.com/keras-team/keras/pull/10673

### CODE REVIEW

**Comment by brge17** [comment]
I signed it.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by brge17** [comment]
Looks like this was fixed internally.
